pymc3/step_methods/slicer.py

```diff
@@ -34,8 +34,8 @@ def __init__(self, vars=None, w=1., tune=True, model=None, **kwargs):
         self.model = modelcontext(model)
         self.w = w
         self.tune = tune
-        self.w_sum = 0
-        self.n_tunes = 0
+        # self.w_sum = 0 #probably we don't need it now
```

Comment:
i'd rather you just delete this (and line 48) -- tests should fail if they are needed.